packages/databricks-vscode/resources/whats-new/1.2.md

```diff
@@ -0,0 +1,48 @@
+# <!-- DATABRICKS LOGO --> Databricks Extension v1.2
```

Comment:
Missing patch number. Would this be v1.2.0?

Reply:
Should probably be full name: "Databricks extension for Visual Studio Code"

Reply:
Patch number is intentionally missing, since this should show up when users cross the minor boundary, for any patch versions. How about v1.2.x?